tests/README.md

```diff
@@ -0,0 +1,63 @@
+# Tests
+
+These tests should ideally be run for every Pull Request, though due to the need to run `clasp login` before some commands, some tests are not run.
+
+
```

Comment:
How do you run tests?
Most people will come here to figure out how to run test and not configure Travis.

Reply:
👍